resolwe_bio/processes/variant_calling/vc_chemut.yml

```diff
@@ -6,6 +6,8 @@
 - slug: vc-chemut
   name: Variant calling (CheMut)
   requirements:
+    resources:
+      memory: 32768
```

Comment:
Are you sure we need 32 GB? Please test locally how much RAM is really needed (if you can) and how it scale with the sample size or some other dependent variable. If you can't test it locally, ask the infrastructure team to set a memory limit for this process on the server directly. We have the capability to override memory setting. I would try 20 GB next. Maybe that's enough. It's really important to keep this requirement to the minimum as it limits us to scale the number of concurrent jobs. We usually have more CPU then RAM on the worker nodes. Also, it's worth considering if we can reduce the amount of memory in use. Try to identify the command (or script) that uses all this memory and if you can either improve the script or set the input parameters.

Finally, if you have good arguments for extending the memory limit to 32 GB do so and write them in change log and the commit message (you can write more text below the first line in the commit message).